test/integration/targets/connection_paramiko_ssh/aliases

```diff
@@ -1,4 +1,6 @@
 needs/ssh
 shippable/posix/group3
 needs/target/setup_paramiko
+skip/freebsd/12.2
+skip/freebsd/13.0
```

Comment:
It would be nice to include a brief one-line comment explaining why the skip is needed. Something like:

```suggestion
skip/freebsd/12.2  # bcrypt 4+ requires a newer version of rust than is available
skip/freebsd/13.0  # bcrypt 4+ requires a newer version of rust than is available
```